sdk/identity/azure-identity/tests/helpers.py

```diff
@@ -18,31 +18,48 @@
 def build_id_token(
     iss="issuer",
     sub="subject",
-    aud="my_client_id",
+    aud="client-id",
     username="username",
-    tenant_id="tenant id",
+    tenant_id="tenant",
     object_id="object id",
-    exp=None,
-    iat=None,
     **claims
-):  # AAD issues "preferred_username", ADFS issues "upn"
-    return "header.%s.signature" % base64.b64encode(
-        json.dumps(
-            dict(
-                {
-                    "iss": iss,
-                    "sub": sub,
-                    "aud": aud,
-                    "exp": exp or (time.time() + 100),
-                    "iat": iat or time.time(),
-                    "tid": tenant_id,
-                    "oid": object_id,
-                    "preferred_username": username,
-                },
-                **claims
-            )
-        ).encode()
-    ).decode("utf-8")
+):
+    token_claims = id_token_claims(
+        iss=iss, sub=sub, aud=aud, tenant_id=tenant_id, object_id=object_id, preferred_username=username, **claims
+    )
+    jwt_payload = base64.b64encode(json.dumps(token_claims).encode()).decode("utf-8")
+    return "header.{}.signature".format(jwt_payload)
+
+
+def build_adfs_id_token(
+    iss="issuer",
+    sub="subject",
+    aud="client-id",
+    username="username",
+    tenant_id="tenant-id",
```

Comment:
Is there a reason the tenant_id here is "tenant-id" but tenant_id is "tenant" in `build_id_token()`?

Reply:
Not a conscious one. I typically slug fake IDs because real ones are GUIDs and I can imagine something in the stack choking on a space. That's probably why past me dropped the " id" above. Can't say why he didn't slug it instead or do similar to "object id".

While thinking about this, I realized these fake tokens shouldn't contain a tenant ID (or an object ID) at all because real ones never do. So I made some changes here to improve consistency and accuracy. Thanks for the scrutiny!